### ISSUE TITLE
Player sprite in night

### ISSUE THREAD

**Issue by AKryukov92** [LordStephen77/DreamBlocks-Beyond-of-Time#9]
![default](https://user-images.githubusercontent.com/24266616/27263591-0e18c7be-548e-11e7-8a49-e7c906aec167.PNG)
Player sprite is too bright in night. It is expected to be dark too.

**Comment by azekillDIABLO** [comment]
I agree, on the website, a screenshot clearly shows it... it think highlight is also too bright

**Comment by tobyplowy** [comment]
@AKryukov92 can you please fix this

**Comment by LordStephen77** [comment]
@AKryukov92 Darkening the player makes the screen completely black, the night is too dark, you do not see anything, I will probably make it less excuse to allow the player to see little, but without having the screen completely black.
